ZAPD/Main.cpp

```diff
@@ -329,6 +265,23 @@ int main(int argc, char* argv[])
 	return 0;
 }
 
+// Windows doesn't make it easy to get a stack trace from just a signal. So we need to do this messy
+// stuf for just windows.
+int main(int argc, char* argv[])
+{
+#ifdef _MSC_VER
+	__try
+	{
+		return Main(argc, argv);
+	}
+	__except (seh_filter(GetExceptionInformation()))
+	{
+	}
+#else
+	return Main(argc, argv);
+#endif
```

Comment:
Seems like it would be sensible to write this the other way round, like all the others. Also rather inconsistent with the others in using `#ifdef` instead of `defined`.

Reply:
I think its better to keep this as `_MSC_VER` since there might be something that isn't `POSIX` and also isn't MSVC. I can change it to `defined` though.